README.md

```diff
@@ -4,18 +4,18 @@
 [![Live](https://img.shields.io/badge/live-brightgreen.svg)](https://devnet.kentico.com/marketplace)
 [![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-ASK%20NOW-FE7A16.svg?logo=stackoverflow&logoColor=white)](https://stackoverflow.com/tags/kentico)
 
-Do you run have an open-source project related to Kentico or some extension you want to share? Submit a Pull Request to add it to the list!
+Are you developing an open-source project or an extension related to Kentico and want to share it with the developer's community? Submit a Pull Request and share it on the [DevNet marketplace](https://devnet.kentico.com/marketplace)!
 
-Visit the [devnet.kentico.com/marketplace](https://devnet.kentico.com/marketplace) to see all extensions!
+Visit [devnet.kentico.com/marketplace](https://devnet.kentico.com/marketplace) to see all available extensions.
 
-## How to qualify
+## Registration requirements
```

Comment:
I would stick with the old headline - we are already linking it from:
https://devnet.kentico.com/articles/download-mvc-widgets-from-the-new-marketplace?viewmode=0

```suggestion
## How to qualify
```